recipes-bsp/optee-os/files/0001-plat-k3-drivers-Add-secure-proxy-driver-for-communic.patch

```diff
@@ -0,0 +1,460 @@
+From cf20f0a4f77eddf47f278f3d20dc786793542ab4 Mon Sep 17 00:00:00 2001
```

Comment:
I didn't test it yet just bring it up for discussion.

Reply:
Didn't checked it but I notice this patch already acked by Linaro, so I guess this is already upstreamed or under upstreaming... So I believe somebody else had already tested it...

I think this patch is the prerequisite for the next SCI api patch so it must be picked.

Reply:
Sure, if we can pick the patches via upstream, that would be better. But the question is also if there is already a release with the patch included...

Reply:
I am ok with a OPTee version bump as long as it does not generate clearing delta.

Reply:
More important would be a careful check if security fixes need to be picked up - we are starting to put OP-TEE on the critical path now for real.

Reply:
Agree. We have CERT report to refer to, maybe that could be helpful.

Reply:
> Sure, if we can pick the patches via upstream, that would be better. But the question is also if there is already a release with the patch included...

there is no release, it's not under any tags, it's just directly from master which could be to dangerous to bump into that state.

Reply:
> Didn't checked it but I notice this patch already acked by Linaro, so I guess this is already upstreamed or under upstreaming... So I believe somebody else had already tested it...
> 
> I think this patch is the prerequisite for the following SCI api patch so it must be picked.

Yes, TI and me both did testing but not for 3.16.0, I'll test it out to check if all works from upstream.